src/main/java/uk/gov/companieshouse/controller/DissolutionController.java

```diff
@@ -136,4 +137,33 @@ public DissolutionPatchResponse patchDissolutionApplication(
             throw new NotFoundException();
         }
     }
+
+    @PatchMapping("/directors/{director-id}")
```

Comment:
please move into a new DissolutionDirectorController given it's different path from these other functions in here.